### DISCUSSION TITLE
Help deploying O-RAN

### DISCUSSION THREAD

**Discussion by marounsaadeh** [srsran/srsRAN_Project#708] [General Help]
Hello,
can anyone recommend me what is the best approach to create an O-RAN lab with using zmq as radio.
is it better to use the gnb or CU, DU split?

currently i have deployed the RIC, gnodeb, Core (open5gs), UE (srsran_4G with zmq)
i would like also to check how i can setup the SMO and O1/A1 interfaces

thanks for helping!!

**Comment by pgawlowicz** [comment]
Hi, 
O1 is not available in the srsRAN Project.
A1 is between Near RT RIC and Non-RT RIC, so it is outside the scope of the srsRAN Project.

Best,
Piotr
